engine/src/flutter/lib/web_ui/lib/src/engine/canvaskit/surface.dart

```diff
@@ -285,10 +271,14 @@ class Surface extends DisplayCanvas {
       }
     }
 
+    // If we reached here, then either we are forcing a new context, or
+    // the size of the surface has changed so we need to make a new one.
```

Comment:
...or we're creating a surface for the first time?

Reply:
Done.